### PR TITLE
Bump to 1.24

### PR DESCRIPTION
**Pull request by skonto**
Fixes JIRA #

## Proposed Changes

- Now that 1.23 is cut let's move on.
- As in https://github.com/openshift-knative/serverless-operator/pull/1479.

### CODE REVIEW

File: olm-catalog/serverless-operator/index/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ COPY --from=quay.io/operator-framework/opm:latest /bin/opm /bin/opm
 COPY configs /configs
 
 RUN /bin/opm init serverless-operator --default-channel=stable --output yaml >> /configs/index.yaml
-RUN /bin/opm --skip-tls-verify render -o yaml registry.ci.openshift.org/knative/openshift-serverless-v1.22.0:serverless-stop-bundle \
+RUN /bin/opm --skip-tls-verify render -o yaml registry.ci.openshift.org/knative/openshift-serverless-v1.23.0:serverless-stop-bundle \
```
Comment (skonto) [inline olm-catalog/serverless-operator/index/Dockerfile:9]: Right now the image is not built, @mgencur should not this be available due to the release cut?

Comment (mgencur) [reply olm-catalog/serverless-operator/index/Dockerfile:9]: The image will be available once any post-submit job runs (after any PR against 1.23 is merged).

**Comment by skonto** [comment]
```
{  the build serverless-operator-src failed after 20m20s with reason DockerBuildFailed: Dockerfile build strategy has failed.

ing-kafka": error during bulk transfer for copier.request{...b.com/prometheus/procfs/proc_io.go: no space left on device}
```

**Pull request merged**